backend/tests/test_integration_address.py

```diff
@@ -1401,23 +1439,24 @@ def test_missing_address(self, mock):
         self.assertRequestResponse(
             '/service/ou/{}/details/address'.format(unitid),
             [{
-                'address_type': address_class,
-                'name': 'Ukendt',
-                'value': addrid,
-                'error': 'no such address {!r}'.format(addrid),
+                'address_type': {
```

Comment:
Erm, the name of this test is `test_missing_address`, yet it now succeeds. Does it still test a properly missing address?

Reply:
It succeeds, yes, in the same way that the previous one did. We're unable to read the address from DAR but still return _something_ to be rendered in the UI.

Reply:
> It succeeds, yes, in the same way that the previous one did. We're unable to read the address from DAR but still return _something_ to be rendered in the UI.

Yeah, I saw that afterwards; I was a bit confused by it appearing slightly less fail-y.

Reply:
I omitted the `error` part of the response. As far as I could tell, we didn't use it for anything. We log the error as well, so we should be good :slightly_smiling_face: